.travis.yml

```diff
@@ -1,8 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - 6
-  - 7
```

Comment:
This is more theoretical at this time, but if Compass is still shipping with node 7.x, e.g. due to [Electron 1.7.5 being node 7.9.0](https://electron.atom.io/), would it be a good idea to keep testing of node 6 or 7?

For example, is anyone likely to use
http://node.green/#ES2017-features-trailing-commas-in-function-syntax or any other newer node feature that would pass Travis here but not work in Compass?

Reply:
ok let's keep 7 in there.